src/views/v2/month/mobile-events/mobile-day/mobile-event.php

```diff
@@ -15,7 +15,7 @@
  *
  * @see tribe_get_event() For the format of the event object.
  */
-$classes = get_post_class( [ 'tribe-events-calendar-month-mobile-events__mobile-event' ], $event->ID );
+$classes = tribe_get_post_class( [ 'tribe-events-calendar-month-mobile-events__mobile-event' ], $event->ID );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: beb955b1047e66912c5093c4b6c266bc</sub>

Reply:
Ignoring.